scripts/test_real_world_ingestion.py

```
@@ -0,0 +1,885 @@
+            }
+        }
+        
+        # First attempt (simulate timeout by using very short timeout)
+        try:
+            await self.client.post(
+                f"{BASE_URL}/api/v1/ingestion/events",
+                headers=HEADERS,
+                json=event,
+                timeout=0.001  # 1ms timeout - will fail
+            )
+        except:
```

Ruff (0.13.3)

Do not use bare `except`

(E722)